### PR TITLE
hv_check_cpu_utilization: Fix 'Access is denied' error for Win11

### PR DESCRIPTION
**Pull request by menli820**
Win11 can not execute 'reg' related command as permission issue.

ID: 1999008
Signed-off-by: Menghuan Li menli@redhat.com

### CODE REVIEW

**Comment by menli820** [comment]
@YongxueHong  please help to review it when you free, thanks

**Comment by menli820** [comment]
related log：

2021-12-23 02:27:00: C:\Windows\system32>
2021-12-23 02:27:00: D:\SetACL -on "HKEY_LOCAL_MACHINE\SYSTEM\CurrentControlSet\Services\SecurityHealthService" -ot reg -actn setowner -ownr "n:Administrators"
2021-12-23 02:27:01: Processing ACL of: <machine\SYSTEM\CurrentControlSet\Services\SecurityHealthService>
2021-12-23 02:27:01: 
2021-12-23 02:27:01: SetACL finished successfully.
2021-12-23 02:27:01: 
2021-12-23 02:27:01: C:\Windows\system32>
2021-12-23 02:27:01: C:\Windows\system32>
2021-12-23 02:27:01: echo %errorlevel%
2021-12-23 02:27:01: 0
2021-12-23 02:27:01: 
2021-12-23 02:27:01: C:\Windows\system32>
2021-12-23 02:27:01: C:\Windows\system32>
2021-12-23 02:27:01: D:\SetACL -on "HKEY_LOCAL_MACHINE\SYSTEM\CurrentControlSet\Services\SecurityHealthService" -ot reg -actn ace -ace "n:Administrators;p:full"
2021-12-23 02:27:01: Processing ACL of: <machine\SYSTEM\CurrentControlSet\Services\SecurityHealthService>
2021-12-23 02:27:01: 
2021-12-23 02:27:01: SetACL finished successfully.
2021-12-23 02:27:01: 
2021-12-23 02:27:01: C:\Windows\system32>
2021-12-23 02:27:01: C:\Windows\system32>
2021-12-23 02:27:01: echo %errorlevel%
2021-12-23 02:27:01: 0
2021-12-23 02:27:01: 
2021-12-23 02:27:01: C:\Windows\system32>
2021-12-23 02:27:01: C:\Windows\system32>
2021-12-23 02:27:01: reg add "HKLM\SYSTEM\CurrentControlSet\Services\SecurityHealthService" /v Start /d 4 /t REG_DWORD /f
2021-12-23 02:27:01: The operation completed successfully.
2021-12-23 02:27:01:

File: qemu/tests/cfg/hv_check_cpu_utilization.cfg [hunk 1]
```diff
@@ -5,13 +5,17 @@
     master_images_clone = image1
     remove_image_image1 =yes
     nics = ""
-    cdroms = ""
```
Comment (YongxueHong) [inline qemu/tests/cfg/hv_check_cpu_utilization.cfg:8]: Why delete this parameter?

File: qemu/tests/hv_check_cpu_utilization.py [hunk 2]
```diff
@@ -114,6 +116,13 @@ def run(test, params, env):
         _stop_service(test, params, session, service)
 
     # stop windows defender
+    if set_owner_cmd:
```
Comment (YongxueHong) [inline qemu/tests/hv_check_cpu_utilization.py:119]:
```suggestion
    if set_owner_cmd and set_full_control:
```

File: qemu/tests/cfg/hv_check_cpu_utilization.cfg [hunk 3]
```diff
@@ -5,13 +5,17 @@
...
     cpu_model_flags = "hv_crash"
     host_check_times = 900
     host_check_interval = 2
     serives_to_stop = wuauserv bits dosvc SysMain
     service_check_cmd = sc query %s
     service_stop_cmd = sc stop {0} & sc config {0} start=disabled
+    Win11:
+        set_owner_cmd = 'WIN_UTILS:\SetACL -on "HKEY_LOCAL_MACHINE\SYSTEM\CurrentControlSet\Services\'
+        set_owner_cmd += 'SecurityHealthService" -ot reg -actn setowner -ownr "n:Administrators"'
+        set_full_control = 'WIN_UTILS:\SetACL -on "HKEY_LOCAL_MACHINE\SYSTEM\CurrentControlSet\Services\'
```
Comment (YongxueHong) [inline qemu/tests/cfg/hv_check_cpu_utilization.cfg:17]:
```suggestion
        set_full_control_cmd = 'WIN_UTILS:\SetACL -on "HKEY_LOCAL_MACHINE\SYSTEM\CurrentControlSet\Services\'
```

**Comment by YongxueHong** [comment]
Please delete the string `ID: 1999008` from your commit message.

File: qemu/tests/cfg/hv_check_cpu_utilization.cfg (hunk 1 above)
Comment (menli820) [reply qemu/tests/cfg/hv_check_cpu_utilization.cfg:8]: Before this fix ,we don't use WIN_UTILS.iso , so use cdroms = ""  to control it.
As we need use 'SetACL.exe' in WIN_UTILS.iso  for current fix, so delete cdroms = ""   to load WIN_UTILS.iso in qemu command line.

Comment (YongxueHong) [reply qemu/tests/cfg/hv_check_cpu_utilization.cfg:8]: Ok, got it, thanks.

**Comment by menli820** [comment]
lastet log:

2021-12-23 03:25:17: D:\SetACL -on "HKEY_LOCAL_MACHINE\SYSTEM\CurrentControlSet\Services\SecurityHealthService" -ot reg -actn setowner -ownr "n:Administrators"
2021-12-23 03:25:17: Processing ACL of: <machine\SYSTEM\CurrentControlSet\Services\SecurityHealthService>
2021-12-23 03:25:17: 
2021-12-23 03:25:17: SetACL finished successfully.
2021-12-23 03:25:17: 
2021-12-23 03:25:17: C:\Windows\system32>
2021-12-23 03:25:17: C:\Windows\system32>
2021-12-23 03:25:17: echo %errorlevel%
2021-12-23 03:25:17: 0
2021-12-23 03:25:17: 
2021-12-23 03:25:17: C:\Windows\system32>
2021-12-23 03:25:17: C:\Windows\system32>
2021-12-23 03:25:17: D:\SetACL -on "HKEY_LOCAL_MACHINE\SYSTEM\CurrentControlSet\Services\SecurityHealthService" -ot reg -actn ace -ace "n:Administrators;p:full"
2021-12-23 03:25:17: Processing ACL of: <machine\SYSTEM\CurrentControlSet\Services\SecurityHealthService>
2021-12-23 03:25:17: 
2021-12-23 03:25:17: SetACL finished successfully.
2021-12-23 03:25:17: 
2021-12-23 03:25:17: C:\Windows\system32>
2021-12-23 03:25:17: C:\Windows\system32>
2021-12-23 03:25:18: echo %errorlevel%
2021-12-23 03:25:18: 0
2021-12-23 03:25:18: 
2021-12-23 03:25:18: C:\Windows\system32>
2021-12-23 03:25:18: C:\Windows\system32>
2021-12-23 03:25:18: reg add "HKLM\SYSTEM\CurrentControlSet\Services\SecurityHealthService" /v Start /d 4 /t REG_DWORD /f
2021-12-23 03:25:18: The operation completed successfully.

03:23:37 DEBUG|     set_full_control_cmd = WIN_UTILS:\SetACL -on "HKEY_LOCAL_MACHINE\SYSTEM\CurrentControlSet\Services\SecurityHealthService" -ot reg -actn ace -ace "n:Administrators;p:full"
03:23:37 DEBUG|     set_owner_cmd = WIN_UTILS:\SetACL -on "HKEY_LOCAL_MACHINE\SYSTEM\CurrentControlSet\Services\SecurityHealthService" -ot reg -actn setowner -ownr "n:Administrators"

**Review by YongxueHong — APPROVED** [review]
LGTM。

**Pull request merged**